transcriptic/config.py

```diff
@@ -149,7 +153,10 @@ def __init__(
                 )
             self.session.headers["Cookie"] = None
             self.email = email
-            self.token = token
+            if token is not None:
+                self.token = token
+            if bearer_token is not None:
+                self.bearer_token = bearer_token
```

Comment:
if both token and bearer_token are presented, how will they be used? For example, in the test case, an invalid token is present but a valid bearer token is given. It suggests that the bearer auth overrides the token auth.

Would prefer a more explicit warning similar to what happens if both cookie and token are set. Otherwise, there may be strange client errors stemming from a combination of a valid/invalid tokens.

Reply:
As of now, in the tx CLI they are not mutually exclusive, _but_ in web the bearer will supersede the traditional token if both are passed in. If the bearer token is invalid and then user token is valid, a 401 will still be returned. 

I'll make them mutually exclusive here and show a warning like we do with cookies.

Reply:
Changes look good. thanks!